open-apparel-registry/open-apparel-registry
# Add selecting silver map style to embed config

## Overview

Edit embed field config to allow users selecting between default map style and silver at _Embed Deluxe / Custom Embed_ and _Embed+_ level, and update model and API accordingly

Connects #1620

## Demo

![2022-02-10 14 09 57](https://user-images.githubusercontent.com/60887686/153479526-fc422b18-be85-41c7-a53f-f51a6f6f7855.gif)

## Testing Instructions

* Check out this branch and run `./scripts/manage migrate`
* Run `./scripts/server`
* In Incognito (private) mode, navigate to [c2@example.com admin](http://localhost:8081/admin/api/contributor/2/change/) page and log in as c1@example.com and make sure the Embed Level is _Embed Deluxe / Custom Embed_
* In a separate browser session, log in as c2@example.com and navigate to Navigate to [settings](http://localhost:6543/settings) and select the Embed tab.
    - [x] Toggle the map style radio buttons - the base map style should update accordingly
 * Go to the admin page in Incognito, set the Embed Level to _Embed+_
    - [x] Refresh the [settings](http://localhost:6543/settings) and select the Embed tab, the previous map style selection is still applies and can also update
 * Go to the admin page in Incognito, set the Embed Level to _Embed_
    - [x] Refresh the [settings](http://localhost:6543/settings) and select the Embed tab and found no map style settings

## Checklist
